src/applications/gi/components/vet-tec/VetTecHeadingSummary.jsx

```diff
@@ -17,6 +17,15 @@ export const VetTecHeadingSummary = ({ institution }) => (
       <h1>{institution.name}</h1>
       <div>
         <div className="usa-width-one-half medium-6 small-12 column">
+          <IconWithInfo
+            icon="star vads-u-color--gold"
```

Comment:
It seems like the original intent of the `icon` prop was just to specify which FA icon to display without writing the full class name. Appending other classes to it might work here, and it's probably fine because the component's usage is limited to just this module, but I think it would look cleaner to have a prop with its own purpose of modifying the icon's style. I would suggest something like:


```
import classNames from 'classnames';

const IconWithInfo = ({ icon, iconClassName, children }) => {
  ...
  <i className={classNames('fa', `fa-${icon}`, iconClassName)} />
  ...
};

...

<IconWithInfo icon="star" iconClassName="vads-u-color--gold">
```

Reply:
Noted. Changes are in.